cms/tests/test_page.py

```diff
@@ -1044,6 +1044,9 @@ def test_rename_node(self):
         parent.publish('en')
         child = create_page('child', 'nav_playground.html', 'en', slug='child', published=True, parent=parent)
         child.publish('en')
+        grandchild = create_page('grandchild', 'nav_playground.html', 'en', slug='grandchild', published=True,
```

Comment:
Please move the grandchild test to its own method.
Something like: `test_rename_node_alters_descendants`